charts/private/open-webui/values.yaml

```
@@ -0,0 +1,7 @@
+open-webui:
+  ollama:
+    enabled: false
+  pipelines:
+    enabled: false
+  ollamaUrls:
+  - <OLLAMA_URL> # Required: The Ollama server URL.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)